scripts/update-translations.sh

```diff
@@ -33,9 +33,19 @@ echo "Pulling translations from transifex"
 tx pull -f
 
 echo "Gathering all translation strings.  Note that this will probably take a while"
-./manage.py makemessages --all
+./manage.py makemessages --all --ignore 'corehq/apps/app_manager/tests/data/v2_diffs*' --ignore 'node_modules'
```

Comment:
Those are definitely both good to skip, we wouldn't want to pull translation-looking things from there even if it didn't happen to crash, so this is an improvement all around